docs/install/installation.rst

```diff
@@ -248,7 +248,7 @@ Docker
 
 MIVisionX provides developers with docker images for Ubuntu `20.04` / `22.04`. Using docker images developers can quickly prototype and build applications without having to be locked into a single system setup or lose valuable time figuring out the dependencies of the underlying software.
 
-Docker files to build MIVisionX containers and suggested workflow are `available <docker/README.md#mivisionx-docker>`_
+Docker files to build MIVisionX containers and suggested workflow are `available <../how-to/mivisionx-docker.html>`_
```

Comment:
This sentence could use a period.

Reply:
The 20.04` / `22.04`. in the previous paragraph shouldn't be in italics

Reply:
Now rethinking the hyperlink that is being fixed. 
Ideally, the link name should be the same or similar to the destination. 
Perhaps this line could be:

For more information about the Docker files used to build MIVisionX containers and a suggested workflow, see **MIVisionX Docker documentation**. (hyperlink)

Reply:
I have added a period and used Leo's suggested text, oh, and removed the italics